libraries/AP_HAL_ChibiOS/hwdef/YJUAV_A6SE_H743/defaults.parm

```diff
@@ -0,0 +1,13 @@
+# setup the heater temperature to 45 degree
+BRD_HEAT_TARG 45
+
+# turn on the CAN power monitoring(default)
+CAN_P1_DRIVER 1
+BATT_MONITOR 4
+
+# setup the parameter for the ADC power module
+BATT_VOLT_MULT 21
```

Comment:
should be in hwdef, and the following line below this...also ou are missing the pin defs for volt and current monitoring